py/desispec/desi_proc_dashboard.py

```diff
@@ -0,0 +1,383 @@
+import argparse
+import os,glob
+import fitsio
+import astropy.io.fits as pyfits
+from astropy.io import fits
+import subprocess
+import pandas as pd
+import time,datetime
+import numpy as np
+import psycopg2
+import hashlib
+import pdb
+import psutil
+from os import listdir
+
+class DESI_PROC_DASHBOARD(object):
+    """ Code to generate the statistic of desi_pipe production status   
+    Usage:
+    Cron job script:
+    */30 * * * * /global/common/software/desi/cori/desiconda/20190804-1.3.0-spec/conda/bin/python3 /global/project/projectdirs/desi/users/zhangkai/desi_code/desi_proc_dashboard.py -n all -nn 60 --prod_dir /global/cscratch1/sd/zhangkai/desi/ --output_dir /global/project/projectdirs/desi/www/users/zhangkai/desi_proc_dashboard/ --output_url https://portal.nersc.gov/project/desi/users/zhangkai/desi_proc_dashboard/  >/global/project/projectdirs/desi/users/zhangkai/desi_proc_dashboard.log 2>/global/project/projectdirs/desi/users/zhangkai/desi_proc_dashboard.err &
+
+    """
+
+    def __init__(self):
+        ############
+        ## Input ###
+        ############
+        parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        parser = self._init_parser(parser)
+        args = parser.parse_args()
+        if not os.getenv('DESI_SPECTRO_REDUX'):
+            os.environ['DESI_SPECTRO_REDUX']='/global/project/projectdirs/desi/spectro/redux/'
+            os.environ['DESI_SPECTRO_DATA']='/global/project/projectdirs/desi/spectro/data/'
```

Comment:
default should be `/global/cfs/cdirs/desi/spectro/redux` and `data` now instead of project.

Reply:
Fixed